src/AnnouncementBubble/NormalAnnouncementBubble.less

```diff
@@ -209,7 +209,6 @@ button.NormalAnnouncementBubble--deleteMenuItem {
 }
 
 .NormalAnnouncementBubble--readReceipts--tooltip {
-  cursor: pointer;
```

Comment:
Don't want the list of names in read receipt tooltip to look clicky since it's not